protocol/config/consensus.go

```diff
@@ -1242,11 +1245,20 @@ func initConsensusProtocols() {
 	// ConsensusFuture is used to test features that are implemented
 	// but not yet released in a production protocol version.
 	vFuture := v38
+
 	vFuture.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 
 	vFuture.LogicSigVersion = 10 // When moving this to a release, put a new higher LogicSigVersion here
 	vFuture.EnableLogicSigCostPooling = true
 
+	vFuture.AgreementDeadlineTimeoutPeriod0 = 4 * time.Second
+
+	vFuture.StateProofBlockHashInLightHeader = true
+
+	// Setting DynamicFilterTimeout in vFuture will impact e2e test performance
+	// by reducing round time. Hence, it is commented out for now.
+	// vFuture.DynamicFilterTimeout = true
+
```

Comment:
Is this stuff supposed to be here?  Seems like it's commenting something that isn't here.

Reply:
It's copy pasted to match what is in go-algorand, expect to change it here in follow-up PR after we make the change in go-algorand as well.

Reply:
oh, I didn't even notice this was sdk